k8s/dialer.go

```diff
@@ -23,7 +23,7 @@ import (
 )
 
 const (
-	socatImage = "quay.io/boson/alpine-socat:1.7.4.3-r1-non-root"
+	socatImage = "quay.io/boson/socat:micro"
```

Comment:
Might be nice to get a version number in here somewhere.

Reply:
+1